addons/godot-next/nodes/callback_delegator.gd

```diff
@@ -0,0 +1,163 @@
+# author: xdgamestudios
+# license: MIT
+# description: Manages a ResourceSet of element resources and delegates Node callbacks to each instance.
+#              As a ResourceSet, only one element of any given type is allowed on a single elements node.
+# deps:
+# - ResourceSet
+# - PropertyInfo
+# - element
```

Comment:
Not sure why this is here. `element` is a script or script class anywhere. It requires that the user define one, but it's a preloaded or global Script/PackedScene dependency, etc.

Reply:
this was the result of using Find/Replace ... sorry